### PR TITLE
Add newsletter signup form to homepage

### LINKED ISSUE #1494: Add newsletter sign up box on the codebar website homepage
## Description of the issue 📄
Currently, the only way to subscribe to the codebar Newsletter is to create an account, we want to change this to have a "Sign up to our Newsletter" section on the homepage of our website.

👀  https://mailchimp.com/help/add-a-signup-form-to-your-website/

## To do 📋

* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

### PR DESCRIPTION
**Pull request by matyikriszta**
## Description
This PR adds a newsletter signup form on the homepage that subscribes people to the main codebar Mailchimp audience. It uses Mailchimps embedded signup form.

## Status
Work in Progress

## TODO
@KimberleyCook @despo there's a piece of copy that needs to be written that explains what to expect from the newsletter, let me know if you have a good idea for this. 1 or 2 short sentences would do.

## Related Github issue
Resolves #1494

## Screenshots
Screenshot 2021-03-30 at 7 13 29 pm

### CODE REVIEW

File: app/views/shared/_newsletter.html.haml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.row
+  .medium-10.medium-offset-1.columns
+    %h2 Subscribe to our newsletter
+    %p Summary of what's in the newsletter. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+    -# Mailchimp Signup Form
+    #mc_embed_signup
```
Comment (despo) [inline app/views/shared/_newsletter.html.haml:7]: Can we use a surrounding container class to restrict the size of the input field? e.g. `medium-4` ?

File: app/views/shared/_newsletter.html.haml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.row
+  .medium-10.medium-offset-1.columns
+    %h2 Subscribe to our newsletter
+    %p Summary of what's in the newsletter. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
[the remark this answers is not in the document]
Comment (matyikriszta) [reply app/views/shared/_newsletter.html.haml:4]: @despo yes, do you have any suggestions for the copy? As mentioned in the PR description I need a good description for what's in the newsletter so people know why it's worth signing up for.

File: app/views/shared/_newsletter.html.haml (hunk 1 above)
Comment (matyikriszta) [reply app/views/shared/_newsletter.html.haml:7]: @despo I tried this but I didn't really like how it looked. Let me do it again and paste a screenshot, we can decide from there.

File: app/views/shared/_newsletter.html.haml (hunk 2 above)
[the remark this answers is not in the document]
Comment (despo) [reply app/views/shared/_newsletter.html.haml:4]: I may have misesd reading the description. Sorry!

[the remark this answers is not in the document]
Comment (despo) [reply app/views/shared/_newsletter.html.haml:4]: > Get monthly updates about the codebar communit. Be informed about upcoming events, jobs and other learning opportunities.

not sure I like the last couple of words... I was thinking the scholarships that we are currently offering for example.

[the remark this answers is not in the document]
Comment (matyikriszta) [reply app/views/shared/_newsletter.html.haml:4]: @despo I think that sounds good, I reworded it slightly to: Get regular updates about the codebar community. Be informed about upcoming events, jobs, scholarships and other learning opportunities. I replaced monthly to regular (the newsletter actually goes out every other month) and added scholarships as a separate listed content item.

File: app/views/shared/_newsletter.html.haml (hunk 1 above)
Comment (matyikriszta) [reply app/views/shared/_newsletter.html.haml:7]: @despo how about this? Also, I just realized we will have to convert this section to bootstrap as part of the migration, we can revisit the design then if we are not satisfied with how this section looks (which I am not). For now I just want to put something out there so people can subscribe.

Screenshot 2021-04-03 at 10 41 09 pm

**Review by despo — APPROVED** [review]
👍🏼

**Pull request merged**